hack/generator/pkg/astmodel/comments.go

```diff
@@ -0,0 +1,102 @@
+package astmodel
+
+import (
+	"go/ast"
+	"strings"
+)
+
+// Utility methods for adding comments
+
+func addDocComment(commentList *[]*ast.Comment, comment string, width int) {
+	for _, c := range formatDocComment(comment, width) {
+		line := strings.TrimSpace(c)
+		if !strings.HasPrefix(line, "//") {
+			line = "//"+line
```

Comment:
Is the reason we're doing `//test` rather than `// text` because of controller-tools behavior with comments and how it renders them?

Might be worth raising an issue there if there already isn't one?

Reply:
I thought @Porges had already done that - I'll leave this open for him to confirm.

Reply:
I have not 🙂